viewer/packages/360-images/src/collection/DefaultImage360Collection.ts

```diff
@@ -44,6 +46,7 @@ export class DefaultImage360Collection implements Image360Collection {
   private readonly _icons: IconCollection;
   private _isCollectionVisible: boolean;
   private readonly _collectionId: string;
+  private readonly _styleAssignmentTracker: AnnotationStyleTracker = new AnnotationStyleTracker();
```

Comment:
Any reason to not initialize this is the constructor?

Reply:
Outdated, removed `AnnotationStyleTracker`